tools/run_tests/run_interop_tests.py

```diff
@@ -288,22 +291,54 @@ def bash_login_cmdline(cmdline):
   return ['bash', '-l', '-c', ' '.join(cmdline)]
 
 
-def cloud_to_prod_jobspec(language, test_case, docker_image=None):
+def add_auth_options(language, test_case, cmdline, env):
+  """Returns (cmdline, env) tuple with cloud_to_prod_auth test options."""
+
+  language = str(language)
+  cmdline = list(cmdline)
+  env = env.copy()
+
+  # TODO(jtattermusch): this file path only works inside docker
+  key_filepath = '/root/service_account/stubbyCloudTestingTest-ee3fce360ac5.json'
```

Comment:
Fine to leave this to be addressed later? I already have a TODO to revisit this (because like this it doesn't work if you are trying to locally and you choose not to use docker).
The expansion of env variable here is a bit tricky, otherwise I would just use $HOME (which would work both with and without docker).